modules/administration-guide/pages/configuring-fuse.adoc

```diff
@@ -0,0 +1,131 @@
+
+This page describes how to make `/dev/fuse` be accessible to workspace containers, which is a requirement for using fuse-overlayfs.
+
+== Enabling container access to /dev/fuse for OpenShift
+
+[WARNING]
+====
+Creating `MachineConfig` resources on an OpenShift cluster is a potentially dangerous task, as it allows making advanced, system-level changes to the cluster.
+
+View the link:https://docs.openshift.com/container-platform/{ocp4-ver}/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[MachineConfig documentation] for more details and possible risks.
+
+====
+
+.Prerequisites
+
+* The link:https://docs.openshift.com/container-platform/{ocp4-ver}/installing/install_config/installing-customizing.html#installation-special-config-butane-install_installing-customizing[Butane] tool (`butane`) is installed in the operating system you are using.
+
+* An active `{orch-cli}` session with administrative permissions to the destination OpenShift cluster. See {orch-cli-link}.
+
+.Procedure
+
+. Set an environment variable depending on whether your OpenShift cluster is a single node cluster, or a multi node cluster with separate control plane and worker nodes.
++
+For a single node cluster, set:
+[subs="+quotes,+attributes,+macros"]
+----
+$ NODE_ROLE=master
+----
+For a multi node cluster, set:
+[subs="+quotes,+attributes,+macros"]
+----
+$ NODE_ROLE=worker
+----
+
+. Create a `MachineConfig` resource that creates a drop-in CRI-O configuration file named `99-podman-fuse` in the `NODE_ROLE` nodes. This configuration file allows access to the `/dev/fuse` device for certain pods.
++
+[subs="+quotes,+attributes,+macros"]
+----
+cat << EOF | butane | oc apply -f -
+variant: openshift
+version: 4.12.0
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${NODE_ROLE}
+  name: 99-podman-dev-fuse-${NODE_ROLE}
+storage:
+  files:
+  - path: /etc/crio/crio.conf.d/99-podman-fuse <1>
+    mode: 0644
+    overwrite: true
+    contents: <2>
+      inline: |
+        [crio.runtime.workloads.podman-fuse] <3>
+        activation_annotation = "io.openshift.podman-fuse" <4>
+        allowed_annotations = [
+          "io.kubernetes.cri-o.Devices" <5>
+        ]
+        [crio.runtime]
+        allowed_devices = ["/dev/fuse"] <6>
+EOF
+----
+<1> The absolute file path to the new drop-in configuration file for CRI-O.
+<2> The content of the new drop-in configuration file.
+<3> Define a `podman-fuse` workload.
+<4> The pod annotation that activates the `podman-fuse` workload settings.
+<5> List of annotations the `podman-fuse` workload is allowed to process.
+<6> List of devices on the host that a user can specify with the `io.kubernetes.cri-o.Devices` annotation.
+
+. After applying the `MachineConfig` resource, scheduling will be temporarily disabled for each node with the `worker` role as changes are applied. View the nodes' statuses.
++
```

Comment:
I'm having trouble with the indentation for this section, as the numbering for the next step resets to 1 (red circle):
![Screenshot from 2024-02-15 10-43-23-edit](https://github.com/eclipse-che/che-docs/assets/83611742/d8892c39-bd35-46db-ae6f-5c7a6f55d4bb)

Any ideas on how I can fix it @deerskindoll ?

Reply:
let's see how the formatting looks with added `+` in the empty lines

Reply:
I've applied the changes, however it is still having issues:
![Screenshot from 2024-02-16 11-21-23](https://github.com/eclipse-che/che-docs/assets/83611742/db083bb9-477d-4f29-a62a-13078a49253d)

Reply:
got it, I'll look into it